sdk/communication/Azure.Communication.CallingServer/src/Azure.Communication.CallingServer.csproj

```diff
@@ -27,7 +27,8 @@
     <Compile Include="$(AzureCoreSharedSources)CancellationHelper.cs" LinkBase="Shared" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Core" />
+    <!-- <PackageReference Include="Azure.Core" /> -->
```

Comment:
Same as comment for sdk/communication/Azure.Communication.CallAutomation/src/Azure.Communication.CallAutomation.csproj